packages/models/tools/generate-types/index.ts

```diff
@@ -0,0 +1,15 @@
+import { join } from 'node:path';
+import { coreConfigSchema, reportSchema } from '../../src';
+import { generateMdFile, generateTypesString, safeWriteFile } from './utils';
+
+const nodeString = generateTypesString({
+  coreConfigSchema,
+  reportSchema,
+});
+// eslint disable-next-line
+// eslint-disable-next-line @typescript-eslint/no-unsafe-call,@typescript-eslint/no-unsafe-member-access,@typescript-eslint/no-unsafe-assignment
+const destDir: string = join(process.cwd(), '..', 'docs');
+// eslint-disable-next-line @typescript-eslint/no-unsafe-call,@typescript-eslint/no-unsafe-assignment
```

Comment:
Isn't everything in `tools` ignored by ESLint? Why so many disable comments?

Reply:
Good point! I checked and somehow I don't get it. the ignore pattern is only working on lib level... hm

Reply:
So it is because we use ignorePatterns also in the nested config. 

If I on the other hand remove it in the nested and put it at root i get another warning:
<img width="887" alt="Screenshot 2024-02-01 at 22 14 47" src="https://github.com/code-pushup/cli/assets/10064416/7c7c55de-b54b-4a3f-9729-f1caefb48bdb">

I could switch to .ts and merge it in `ts`or configure it in the models `eslintrc.json`.